docs/modules/clients/pages/clc.adoc

```diff
@@ -0,0 +1,350 @@
+* [optional] --value-type/-t: specifies the type of the value
+**   possible values: string [default], boolean, json, int8, int16, int32, int64, float32, float64
+* --max-idle: the maximum time for this entry to stay idle in the map
+** possible values: ns, us, ms, s, m, h (nanosecond, microsecond, millisecond, second, minute, hour). Can not be shorter than 1 second.
+* --ttl: duration after which entry will expire and get evicted
+** possible values: ns, us, ms, s, m, h (nanosecond, microsecond, millisecond, second, minute, hour). Can not be shorter than 1 second.
+
+`hzc map put --key-type string --key hello --value-type float32 --value 19.94 --name myMap --ttl 1300ms --max-idle 1400ms`
+
+*put-all*: Puts given key-value pairs as entries to the specified map.
+
+* --key/-k: key of the entry
+* --name/-n: name of the map
+* --value/-v: value to be set
+** mutually exclusive with "--value-file", "--json-entry"
+* --value-file/-f: path to file that contains the value to be set. Use "-" (dash) to read from stdin
+** mutually exclusive with "--value", "--json-entry"
+* [optional] --key-type: specifies the type of the key
+** possible values: string [default], boolean, json, int8, int16, int32, int64, float32, float64
+* [optional] --value-type/-t: specifies the type of the value
+** possible values: string [default], boolean, json, int8, int16, int32, int64, float32, float64
+** mutually exclusive with "--json-entry"
+* --json-entry: path to json file that contains entries
+** mutually exclusive with "--value", "--value-file", "--value-type"
+
+[source,bash,subs="attributes+"]
+----
+# Keys and values are matched with the given order
+hzc map put-all -n mapname --key-type int16 -k 1 -k 2 --value-type json -f valueFile.json -v '{"field":"tmp"}'
+hzc map put-all -n mapname --json-entry entries.json`
+----
+
+Example json-entry file:
+```
+{
+     "key1": "value1",
+     "key2": {
+       "innerData": "data",
+       "anotherInnerData": 5.0
+     },
+     "key3": true,
+     "key4": [1, 2, 3, 4, 5]
+   }
+```
+
+*remove*: Deletes the value for the given key from the specified map.
+
+* --name/-n: name of the map
+* --key/-k: key of the entry
+* [optional] --key-type: specifies the type of the key
+**   possible values: string [default], boolean, json, int8, int16, int32, int64, float32, float64
+
+` hzc map remove -n mapname -k k1`
+
+*use*: Sets the default map name (interactive-mode only).
+
+* --reset: unset default name for map
+
+[source,bash,subs="attributes+"]
+----
+Available Commands:
+hzc map use m1    # sets the default map name to m1 unless set explicitly
+hzc map get --key k1    # "--name m1" is inferred
+hzc map use --reset	  # resets the behaviour
+----
+
+==== Cluster
+Cluster commands are a group of administrative cluster operation.
+
+Hzc uses REST-API for cluster management operations. In order to perform them, you have to enable the REST-API in the "member"
```

Comment:
```suggestion
You must enable the REST API on all members of your cluster to use the cluster commands. These commands use the Hazelcast REST API for cluster management operations. To enable REST API, follow the instructions in the member log or see the documentation: xref:maintain-cluster:rest-api.adoc#enabling-rest-api[enabling-rest-api].
```

Paragraphs should follow the what, why, how structure.

Reply:
Thanks for the tip! https://github.com/hazelcast/hz-docs/pull/462/commits/b3e94f20a0864ac722d6b633d8dc5a57e9758e7b